oomcli/cmd/get_meta_entity.go

```diff
@@ -14,28 +14,33 @@ import (
 	"github.com/spf13/cobra"
 )
 
-var listEntityCmd = &cobra.Command{
+var getMetaEntityCmd = &cobra.Command{
 	Use:   "entity",
-	Short: "list all existing entities",
+	Short: "get existing entity given specific confitions",
```

Comment:
add `Args` to check it could have at most 1 argument

Reply:
done